easybuild/easyconfigs/d/davix/davix-0.6.6-intel-2017a.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'CMakeMake'
+
+name = 'davix'
+version = '0.6.6'
+
+homepage = 'http://dmc.web.cern.ch/projects/davix/home'
+description = """The davix project aims to make file management over HTTP-based protocols simple.
+ The focus is on high-performance remote I/O and data management of large collections of files.
+ Currently, there is support for the WebDav (link is external), Amazon S3 (link is external),
+ Microsoft Azure (link is external), and HTTP (link is external) protocols."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = ['https://github.com/cern-it-sdc-id/davix/archive/']
+sources = ['R_%s.tar.gz' % version.replace('.','_')]
+
+builddependencies = [
+    ('CMake', '3.8.1'),
+    ('LibUUID', '1.0.3'),
+    ('gtest', '1.8.0'),
+    ('Doxygen', '1.8.13'),
+]
+
+dependencies = [
+    ('libxml2', '2.9.4'),
+    ('gSOAP', '2.8.47'),
+    ('Boost', '1.63.0', '-Python-2.7.13'),
+]
+
+osdependencies = [('openssl-devel', 'libssl-dev', 'libopenssl-devel')]
+
+sanity_check_paths = {
+    'files': [
+        'bin/davix-get',
+        'bin/davix-http',
+        'bin/davix-ls',
+        'bin/davix-mkdir',
+        'bin/davix-mv',
+        'bin/davix-put',
+        'bin/davix-rm'
```

Comment:
@smoors let's collapse this to a single line, or maybe use a list comprehension like:

```python
    'files': ['bin/davis-%s' % x for x for in ['get', 'http', 'ls', 'mkdir', 'mv', 'put', 'rm']],